services/infra/InfraService.ts

```diff
@@ -91,9 +134,20 @@ export default class InfraService {
       await this.sitesService.update(updateSuccessSiteInitParams)
       logger.info(`Successfully created site on Isomer, site ID: ${newSite.id}`)
 
-      // TODO: Handle success by sending email to user who requested to create site
+      await this.sendCreateSuccess(
+        requesterEmail,
+        repoName,
+        submissionId,
+        deployment.stagingUrl,
+        deployment.productionUrl
+      )
```

Comment:
Should this be outside the try block? Otherwise failing at this step will attempt to send a repo creation email instead

Reply:
Yes, that's a bit better. I'll do that.

Reply:
I cant easily move this outside the try/catch block, because the success message depends on information that is in the block’s scope. I could create mutable variable to hold that information and make it available in a wider scope, but that seems uglier than just leaving it the way it is.